examples/web-getting-deployed/src/jvmMain/kotlin/Server.kt

```diff
@@ -0,0 +1,44 @@
+
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.ContentNegotiation
+import io.ktor.html.respondHtml
+import io.ktor.http.content.resource
+import io.ktor.http.content.static
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import kotlinx.html.body
+import kotlinx.html.div
+import kotlinx.html.head
+import kotlinx.html.id
+import kotlinx.html.script
+import kotlinx.html.title
+
+var jsFileName = "web-getting-deployed.js"
+var jsFileMapName = "$jsFileName.map"
+
+fun Application.main() {
+
+    install(ContentNegotiation)
+
+    routing {
+
+        get("/") {
+            call.respondHtml {
+                head {
+                    title("web-getting-deployed")
+                }
+                body {
+                    div { id = "root" } // Required by Compose Web
+                    script(src = "/static/$jsFileName") {}
+                }
+            }
+        }
+
+        static("/static") {
+            resource(jsFileName)
+            resource(jsFileMapName)
```

Comment:
Is there a reason to add jsFileMap here? Is it needed for something to make the app work?

Reply:
It's not strictly required to make the App work; but highly recommended to include in a development project.  The browser will automatically read this accompanying, same-named `.map` file alongside the JS, enabling it to show reference to the original Kotlin source where exceptions occur.